tools/clang/test/CodeGenDXIL/hlsl/intrinsics/IsSpecialFloat6.8.hlsl

```diff
@@ -0,0 +1,32 @@
+// RUN: %dxc -T lib_6_8 -enable-16bit-types %s | FileCheck %s
+
+// CHECK-LABEL: test_isinf
+// CHECK: [[V1:%.*]] = bitcast half {{.*}} to i16
+// CHECK: [[V2:%.*]] = icmp eq i16 [[V1]], 31744
+// CHECK: [[V3:%.*]] = icmp eq i16 [[V1]], -1024
```

Comment:
Might be worth leaving a comment saying these are the float representations of positive and negative inf, since I was confused what the 31744 / -1024 was for.